pkg/tiller/hooks.go

```diff
@@ -137,10 +138,6 @@ func (file *manifestFile) sort(result *result) error {
 			return e
 		}
 
-		if entry.Version != "" && !file.apis.Has(entry.Version) {
```

Comment:
To be really clear: I am removing this set of checks. It appears to duplicate the check in validateManifest, and it fires well before we have a chance to install the CRD.